plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/webview/Browser.kt

```diff
@@ -148,6 +148,7 @@ class Browser(parent: Disposable, private val webUri: URI, val project: Project)
                         {
                         agenticMode: true,
                         quickActionCommands: [],
+                        modelSelectionEnabled: ${isModelSelectionEnabled()},
```

Comment:
if required, hardcode to true because we do not want a dependency on server to be ready before ui can be created